### ISSUE TITLE
Pegar o codigo token?

### ISSUE THREAD

**Issue by Evandromsousa** [andreroggeri/pynubank#393]
Queria saber se tem como após fazer autenticação cpf+senha+certificado é possivel retornar o valor do token
para poder fazer uma requisição na api através do codigo token no bearer

**Comment by andreroggeri** [comment]
Sim, o método de autenticação retorna um refresh token que pode utilizado no `authenticate_with_refresh_token`
